### PR TITLE
doc: Misc RPC help fixes

### PR DESCRIPTION
**Pull request by maflcko**
This fixes documentation of the following RPCs:

* estimaterawfee (hidden)
* https://bitcoincore.org/en/doc/0.19.0/rpc/wallet/walletprocesspsbt/
* https://bitcoincore.org/en/doc/0.19.0/rpc/rawtransactions/finalizepsbt/
* https://bitcoincore.org/en/doc/0.19.0/rpc/network/getnetworkinfo/
* https://bitcoincore.org/en/doc/0.19.0/rpc/wallet/listsinceblock/
* https://bitcoincore.org/en/doc/0.19.0/rpc/wallet/importmulti/

<!-- Also, it comes with a scripted diff to normalize whitespace and type names. (Previous attempts: #14601 and #14459)

### CODE REVIEW

**Comment by practicalswift** [comment]
Concept ACK: nice cleanups!

File: src/rpc/blockchain.cpp [hunk 1]
```diff
@@ -1204,39 +1204,39 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
...
-            "  \"prune_target_size\": xxxxxx,  (numeric) the target size used by pruning (only present if automatic pruning is enabled)\n"
-            "  \"softforks\": {                (object) status of softforks\n"
+            "  \"chain\" : \"xxxx\",              (string) current network name (main, test, regtest)\n"
+            "  \"blocks\" : xxxxxx,             (numeric) the height of the most-work fully-validated chain. The genesis block has height 0\n"
+            "  \"headers\" : xxxxxx,            (numeric) the current number of headers we have validated\n"
+            "  \"bestblockhash\" : \"...\",       (string) the hash of the currently best block\n"
+            "  \"difficulty\" : xxxxxx,         (numeric) the current difficulty\n"
+            "  \"mediantime\" : xxxxxx,         (numeric) median time for the current best block\n"
+            "  \"verificationprogress\" : xxxx, (numeric) estimate of verification progress [0..1]\n"
+            "  \"initialblockdownload\" : xxxx, (boolean) (debug information) estimate of whether this node is in Initial Block Download mode.\n"
```
Comment (kristapsk) [inline src/rpc/blockchain.cpp]: Maybe change "xxxx" to "true|false" here and elsewhere where it's boolean too, as it is in other places?

File: src/rpc/blockchain.cpp [hunk 2]
```diff
@@ -1204,39 +1204,39 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
...
-            "  \"chainwork\": \"xxxx\"           (string) total amount of work in active chain, in hexadecimal\n"
-            "  \"size_on_disk\": xxxxxx,       (numeric) the estimated size of the block and undo files on disk\n"
-            "  \"pruned\": xx,                 (boolean) if the blocks are subject to pruning\n"
-            "  \"pruneheight\": xxxxxx,        (numeric) lowest-height complete block stored (only present if pruning is enabled)\n"
-            "  \"automatic_pruning\": xx,      (boolean) whether automatic pruning is enabled (only present if pruning is enabled)\n"
-            "  \"prune_target_size\": xxxxxx,  (numeric) the target size used by pruning (only present if automatic pruning is enabled)\n"
-            "  \"softforks\": {                (object) status of softforks\n"
+            "  \"chain\" : \"xxxx\",              (string) current network name (main, test, regtest)\n"
+            "  \"blocks\" : xxxxxx,             (numeric) the height of the most-work fully-validated chain. The genesis block has height 0\n"
```
Comment (kristapsk) [inline src/rpc/blockchain.cpp]: Numeric uses "n" for numbers and "ttt" for timestamps in other places, maybe change others too instead of "xxx"?

Comment (MarcoFalke) [reply src/rpc/blockchain.cpp]: Actually, most of the times are denoted with `xxx`, not `ttt`. Anyway, this is fixed in #17809

File: src/rpc/blockchain.cpp (hunk 1 above)
Comment (MarcoFalke) [reply src/rpc/blockchain.cpp]: Done in #17809

File: src/rpc/blockchain.cpp (hunk 2 above)
Comment (promag) [reply src/rpc/blockchain.cpp]: Out of curiosity, why is xxxxxx (as well the other "values") necessary? Isn't the type sufficient ?

File: src/rpc/mining.cpp [hunk 3]
```diff
@@ -154,7 +154,7 @@ static UniValue generatetodescriptor(const JSONRPCRequest& request)
             {"maxtries", RPCArg::Type::NUM, /* default */ "1000000", "How many iterations to try."},
         },
         RPCResult{
-            "[ blockhashes ]     (array) hashes of blocks generated\n"},
+            "[ blockhashes ]     (json array) hashes of blocks generated\n"},
```
Comment (promag) [inline src/rpc/mining.cpp]: 👀 this is json-rpc?

**Review by promag — COMMENTED** [review]
Concept ACK.

File: src/rpc/blockchain.cpp (hunk 2 above)
Comment (MarcoFalke) [reply src/rpc/blockchain.cpp]: It serves a placeholder for the value in the key-value pair of a json dict

File: src/rpc/mining.cpp (hunk 3 above)
Comment (MarcoFalke) [reply src/rpc/mining.cpp]: Yes, RPC arguments are called this way. Also, most of the RPC results are called this way. This adjusts the remaining ones.

**Comment by fjahr** [comment]
ACK fa8f302c6354bb41c2e7cbdb3448cc1b91c9a531

Reviewed code and did some selective testing.

**Comment by MarcoFalke** [comment]
Rebased due to conflict in scripted-diff. Should be easy to re-ACK

**Comment by emilengler** [comment]
> This fixes documentation of the following RPCs:

Are they generated automatically? If so how?
Would like to test it

**Comment by MarcoFalke** [comment]
> Are they generated automatically? If so how?

 https://github.com/bitcoin-core/bitcoincore.org/tree/master/contrib#rpc-documentation-generation

**Comment by MarcoFalke** [comment]
Resolved merge conflict

**Comment by laanwj** [comment]
ACK fa5c6622c8ecf1954e7177888ad8c97a77b16fb7

**Pull request merged**